patches/minecraft/net/minecraft/core/RegistryAccess.java.patch

```diff
@@ -1,11 +1,36 @@
 --- a/net/minecraft/core/RegistryAccess.java
 +++ b/net/minecraft/core/RegistryAccess.java
+@@ -37,7 +_,7 @@
+ 
+ public abstract class RegistryAccess {
+    static final Logger f_123047_ = LogManager.getLogger();
+-   static final Map<ResourceKey<? extends Registry<?>>, RegistryAccess.RegistryData<?>> f_123048_ = Util.m_137537_(() -> {
++   static final Map<ResourceKey<? extends Registry<?>>, RegistryAccess.RegistryData<?>> f_123048_ = new java.util.HashMap<>(Util.m_137537_(() -> {
```

Comment:
I would usually avoid making immutable maps mutable and settle for some joined forge variant, but since this is inaccessible normally, this is more of an opinion.

Reply:
Yeah I had similar feelings but the access swayed it for me, personally. This map is used when statically initializing the BUILTIN registry holder so I think it'd become complicated to have forge inject in the REGISTRIES lambda, for example, at an apropriate time - it'd be frail to classloading.